.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# these owners will be requested for review when someone
+# opens a pull request.
+*       @dav3r @felddy @hillaryj @jsf9k @mcdonnnj @cisagov/team-ois
```

Comment:
@mcdonnnj - please remove @hillaryj and @jsf9k as code owners, as part of our effort to be more targeted with owners.

Reply:
Should we also remove `@cisagov/team-ois`?

Reply:
Yes, please remove `@cisagov/team-ois` too - thanks!

Reply:
👍 Trimmed in 4480be7